server/src/internalClusterTest/java/org/opensearch/action/admin/cluster/shards/TransportCatShardsActionIT.java

```diff
@@ -125,4 +128,33 @@ public void onFailure(Exception e) {
         latch.await();
     }
 
+    public void testCatShardsSuccessWithPaginationWithClosedIndices() throws InterruptedException, ExecutionException {
```

Comment:
- Can we include closed indices across pages and verify that all the required fields are present in the output ?
- might be also good to add a test that does `/_cat/shards` call and ensures that the response match.

Reply:
Expanded the test to make both cat/shards and list/shards call, and verify that the number of shardRoutings to be displayed and number of stats fetched are same for both the calls.